docs/docs/build/metrics-view/what-are-metrics-views.md

```diff
@@ -0,0 +1,129 @@
+
+<div style={{ textAlign: 'center' }}>
+  <img src="/img/concepts/metrics-view/metrics-view-components.png" width="100%" style={{ borderRadius: '15px', padding: '20px' }} />
+</div>
+
+
+In Rill, your metrics view is defined by _metric definitions_. Metric definitions are composed of:
+* _**model**_ - A data model creating a One Big Table that will power the metrics view.
+* _**timeseries**_ - A column from your model that will underlie x-axis data in the line charts. Time will be truncated into different time periods.
+* _**measures**_ - Numerical aggregates of columns from your data model shown on the y-axis of the line charts and the "big number" summaries.
+* _**dimensions**_ - Categorical columns from your data model whose values are shown in _leaderboards_ and allow you to look at segments or attributes of your data (and filter/slice accordingly).
+
+## Creating a Metrics view
+
+When creating a metrics view in Rill, you can either use the visual metrics editor or YAML.
+
+### Using the Visual Metrics Editor
+
+<img src='/img/build/metrics-view/visual-metrics-editor.png' class='rounded-gif' />
+<br />
+
+When you add a metrics definition using the UI, a code definition will automatically be created as a YAML file in your Rill project within the metrics directory by default.
+
+### Using YAML
+
+You can also create metrics definitions directly by creating the artifact yourself.
+
+In your Rill project directory, after the `metrics-view.yaml` file is created in the `metrics` directory, its configuration or definition can be updated as needed by updating the YAML file directly, using the following template as an example:
+
+```yaml
+# Metrics View YAML
+# Reference documentation: https://docs.rilldata.com/reference/project-files/metrics_views
+
+version: 1
+type: metrics_view
+
+model: example_model # Choose a table to underpin your metrics
+timeseries: timestamp_column # Choose a timestamp column (if any) from your table
+
+dimensions:
+  - column: category
+    label: "Category"
+    description: "Description of the dimension"
+
+measures:
+  - expression: "SUM(revenue)"
+    label: "Total Revenue"
+    description: "Total revenue generated"
+```
+
+:::info Check our reference documentation
+
+For more information about available metrics view properties, feel free to check our [reference YAML documentation](/reference/project-files/metrics-views).
+
+:::
+
+### Creating a Metrics View with AI
+
+In order to streamline the process and get to a dashboard as quickly as possible, we've added the "Create Metrics with AI" and "Create Dashboard with AI" options! This will pass your schema to OpenAI to suggest measures and dimensions to get started with Rill. You can define your own OpenAI key by creating a [connector file](/reference/project-files/connectors#ai). If you want to disable AI from your environment, please set the following in the `rill.yaml`:
+
+```yaml
+features:
+  ai: false
+```
+
+## Creating Valid Metrics
+
+### Time series
```

Comment:
I could see there being a standalone Metrics View docs page dedicated to the `timeseries` key and all time-related documentation.